### ISSUE TITLE
instruction "--bootstrap" like PHPUnit

### ISSUE THREAD

**Issue by Darracq** [nette/tester#275]
Maybe it would be a good idea to implement the instruction "--bootstrap" like PHPUnit

```
phpunit --bootstrap src/autoload.php tests/MoneyTest
```
(from https://phpunit.de/getting-started.html)

With this we avoid writing in each *.phpt file lines like:
```
require __DIR__ . '/bootstrap.php';
//  if you want have the same folder structure like your '/app':
require __DIR__ . '/../../../../../bootstrap.php';
```

It also allows you to use different "bootstrap.php" if you need testing in different environments.
```
tester --bootstrap 'bootstrap_test_server.php'
tester --bootstrap 'bootstrap_local_server.php'
```

**Comment by milo** [comment]
Tester is different. Every test behaves as an ordinary PHP file and you can run it as:
```php
php tests/dir/test.phpt
```
Or you can easily debug it and step through it IDE.

`vendor/bin/tester` is only tests runner. It takes every single test and run it as a standalone process.

When you consider this, some kind of bootstrapping is necessary.

I have one workaround on my mind... you can try
`vendor/bin/tester -d include_path=/dir/with/bootstrap tests`

Basic tutorial is at https://tester.nette.org/

**Comment by milo** [comment]
And next ugly workaroung... set environment variable and use it in tests
```
BOOT=/path/to/bootstrap.php vendor/bin/tester tests
```

**Comment by milo** [comment]
I realized, this feature is already implementeed by PHP itself:
```
vendor/bin/tester -d auto_prepend_file=/path/to/bootstrap.php tests
```

**Comment by Fuco1** [comment]
I would much rather not use php language flags, they are pretty unreliable, depend on ini settings and god knows what else. Having this feature is as simple as 

```php
		$autoloadFile = $input->getOption('autoload-file');
		if ($autoloadFile !== null && is_file($autoloadFile)) {
			$autoloadFile = $fileHelper->normalizePath($autoloadFile);
			if (is_file($autoloadFile)) {
				require_once $autoloadFile;
			}
		}
``` 

(code from [phpstan](https://github.com/phpstan/phpstan/blob/master/src/Command/AnalyseCommand.php#L69))

Is there really any drawback of having such an option?  It would save a lot of useless `require_once` in tests.

**Comment by milo** [comment]
No drawbacks. I just don't see any advantage.

**Comment by Fuco1** [comment]
When you refactor your hierarchy or move tests around the `require_once` paths can/will break and this is an unnecessary headache.  This happened to me at my last job and it was a pain.  Also I suppose I just find it ugly, but this is down to personal preference.

Having things set up once in a shell script or whatever build system is simpler and futureproof.

**Comment by Fuco1** [comment]
I don't quite understand how `--bootstrap` is different, maybe it's the same thing and I just don't see it.

I understand 1., it may become a maintenance nightmare answering the same question all over.

2. I imagined it would simply include a file I provided.  The file would require autoloader (if any), call `setup()` and whatever else I need.  So I see it that `--bootstrap` is some sort of automagic way to do that?
